parameter_expansion/tests/test_pe.py

```diff
@@ -79,6 +79,78 @@
     ),
 }
 
+substring_test_cases = {
+    # string,               parameter, result
+    "-${parameter:0:2}-": (
+        "aa/bb/cc",
+        "-aa-",
+    ),
+    "-${parameter:3:2}-": (
+        "aa/bb/cc",
+        "-bb-",
+    ),
+    "-${parameter:6}-": (
+        "aa/bb/cc",
+        "-cc-",
+    ),
+}
+
+replace_test_cases = {
+    # string,               parameter, result
+    "-${parameter/aa/bb}-": (
+        "aa/bb/cc",
+        "-bb/bb/cc-",
+    ),
+    "-${parameter/aa/}-": (
```

Comment:
If you implement

```bash
$ parameter=aa/bb/cc; echo "-${parameter/aa}-"
-/bb/cc-
```

please add a test case for it.

Reply:
Done. Thanks

Reply:
I also added a case for:
```
$ parameter=aa/bb/aa; echo "-${parameter//aa}-"
-/bb/-
```